### PR TITLE
client, templates: Make image cutoff a little better by truncating text

### LINKED ISSUE #47: Name field needs to be truncated to a set length
Files with long names mess up the image positioning when viewing with tags/name.
Probably should truncate the name with JS or in qtpl to a set limit so the image isn't way off to the side.

### PR DESCRIPTION
**Pull request by ghost**
Fixes #47
Unfortunately, I don't think this is the only thing that needs to be done, we probably need a flex bar for the width so the image doesn't get cut off anyway, so we can just scroll to see the rest.
Either that, or resize the image HTML to fit the left-side panel.

### CODE REVIEW

File: templates/image.qtpl [hunk 1]
```diff
@@ -21,7 +21,7 @@
 		<div id="image-view">
 			<section id="tags">
 				{% if img.Name != "" %}
-					<span class="image-name">
+					<span class="spaced truncate image-name">
```
Comment (bakape) [inline templates/image.qtpl]: What is the `spaced` for?

Comment (ghost) [reply templates/image.qtpl]: No idea.
I guess I'll remove it.

**Comment by bakape** [comment]
>we probably need a flex bar for the width so the image doesn't get cut off anyway

Why? I'd think that max-width takes care of that.

File: templates/image.qtpl (hunk 1 above)
Comment (bakape) [reply templates/image.qtpl]: Sasuga.

**Comment by ghost** [comment]
Because the image gets cut-off regardless.

**Comment by bakape** [comment]
Doesn't sound right. I'll investigate this later.

**Comment by ghost** [comment]
Yup. Gonna move jshint to devDependencies too.
No need for that to be in the main.

**Comment by bakape** [comment]
Works on my machine™. What was the problem with this again?

**Comment by ghost** [comment]
I don't remember, if it comes up, I'll probably do another PR.

**Pull request merged**